storybook/cypress/e2e/logo.cy.ts

```diff
@@ -0,0 +1,35 @@
+describe("Logo", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:45000/iframe.html?id=core-logo--default");
+  });
+
+  it("shows a label", () => {
+    cy.get("dso-logo")
+      .invoke("attr", "label", "Beheerportaal")
+      .shadow()
+      .find(".logo-content-label")
+      .should("be.visible");
+  });
+
+  it("only shows a label on smaller screens", () => {
+    cy.viewport(400, 720);
+    cy.get("dso-logo")
+      .invoke("attr", "label", "Beheerportaal")
+      .shadow()
+      .find(".logo-content-label")
+      .should("be.visible")
+      .find(".logo-omgevingsloket")
+      .should("not.exist");
+  });
+
+  it("shows the beta tag", () => {
+    cy.get("dso-logo").invoke("attr", "ribbon", "beta").shadow().find(".logo-ribbon").should("be.visible");
```

Comment:
* Nu test je alleen op zichtbaarheid, maar je kan ook testen of de waarde wel echt in het element is gezet.
* Overwogen om de functionaliteit in `aria-label` te testen?

Reply:
Ik heb de test aangepast, maar ik weet niet wat je bedoelt met de functionaliteit in aria-label. (Heb ik handmatig wel gechecked)

Reply:
> [...] ik weet niet wat je bedoelt met de functionaliteit in aria-label.

De verwachting is dat `label` en `ribbon` in een bepaald format in `aria-label` worden gezet.

> (Heb ik handmatig wel gechecked)

En dat kunnen we ook automatisch doen 😄